src/App.js

```diff
@@ -1,11 +1,46 @@
 import "./App.css";
 import React from "react";
 import Form from "./components/Form";
+import ProctectedRoute from "./components/ProctedRoutes";
+import useAuth from "./components/UseAuth";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import Register from "./components/Register";
+import Login from "./components/Login";
 
 function App() {
+  const [isAuth, login, logout] = useAuth(false);
   return (
-    <div>
-      <Form />
+    <div className="container">
+      <h1>Todo List</h1>
+      <Router>
+        <Switch>
+          <Route path="/" exact component={Login} />
+        </Switch>
+        <Switch>
+          <Route path="/signup" component={Register} />
+        </Switch>
+        <Switch>
+          <ProctectedRoute path="/form" component={Form} auth={isAuth} />
+        </Switch>
```

Comment:
Just need to use `<Switch>` as a wrapper. No need to use it with every single route.

Reply:
Done